packages/terra-show-hide/tests/jest/ShowHide.test.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
```

Comment:
You can now import `IntlProvider` from [terra-base](https://github.com/cerner/terra-core/blob/master/packages/terra-base/src/Base.jsx#L109) which should allow you to drop the react-intl devDependency.

Reply:
That release just happened on October 2nd. Maybe try again after a fresh install?

Reply:
the install fixed the unresolved variable error. but it's still throwing an error every time I try to render: "Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: %s.%s%s". Still working on it

Reply:
I got it working on my pr with version 3.24.0

Reply:
After I figured out which pr @JakeLaCombe was referring to, I removed the renderer function and used IntlProvider the way he designed it here: https://github.com/cerner/terra-framework/blob/070e56a73fbff9b9a29b61bf05581695d321d5a8/packages/terra-application-header-layout/tests/jest/ApplicationHeader.test.jsx. This method works if I use shallow() but it will not work for render(). If I use shallow, I don't have access to any of the props or attributes passed into the component. I will have to rewrite all of these Jest tests to use  toMatchSnapshot() instead of validating the JSON object. I'll start that rewrite now.

Would it be possible to update the documentation for the developer workflow about when to use injectIntl vs when to use context and the correct standard for testing both of them? Maybe even pointing to an example component that upholds these standards, since there are several components with different implementations. This would save future first-time contributors a lot of time and effort

Reply:
We can leave the import from reac-intl as is, lets not go through rewriting the tests now. 
One of the terra team members can update it in a followup PR. Since we just made the change to export IntlProvider from base a couple places, there will be a PR we have to switch the import from react-intl to terra-base for all components that are pulling in IntlProvider.

Reply:
[Logged an issue to docuent](https://github.com/cerner/terra-ui/issues/119) when to use injectIntl vs when to use context and the correct standard for testing both of them.